### ISSUE TITLE
Bot Restarts When Commanders Tagged Down

### ISSUE THREAD

**Issue by ogrady** [FlussuferOrga/Botgart#52]
Bot consistently restarts after an empty list of commanders was received from TS.
```
2020-03-17T10:54:33.547Z debug [BotgartClient.js]: Done writing WvWStats.
2020-03-17T10:55:29.613Z debug [TS3Listener.js]: Received from TS-Bot: {"commanders":[{"account_name":"LumPii.4195","ts_cluid":"TROAV7d9EyyO90fqIzQcChl5+Bs=","ts_display_name":"Lizzy","ts_channel_name":"Allgemeine Streitkräfte [DE]"}]}
2020-03-17T10:56:29.769Z debug [TS3Listener.js]: Received from TS-Bot: {"commanders":[{"account_name":"LumPii.4195","ts_cluid":"TROAV7d9EyyO90fqIzQcChl5+Bs=","ts_display_name":"Lizzy","ts_channel_name":"Allgemeine Streitkräfte [DE]"}]}
2020-03-17T10:57:39.408Z debug [TS3Listener.js]: Received from TS-Bot: {"commanders":[]}
2020-03-17T10:57:40.700Z info [Botgart.js]: Registered achievement 'Glimmer'.
```

The restart happens before the Commander-role is removed from the formerly active commander, which is evident from every commander retaining their role even after their lead.

This behaviour started after fixing the achievements that are based on match updates. 
I suspect some sort of timeout with the DB to be the issue.

**Comment by ogrady** [comment]
This appears to have been a problem with IPC and seems to be fixed since swapping TCP for REST.

**Comment by ogrady** [comment]
```
botgart_1  | 2020-10-30T17:38:56.178Z info [TS3Connection.js]: Tagging up [DNGO] Tyrone | Butcher in Flussufer.
botgart_1  | 2020-10-30T17:48:59.579Z info [TS3Connection.js]: Tagging up Domi | Chaoskruemel in Flussufer.
botgart_1  | 2020-10-30T17:48:59.582Z info [TS3Connection.js]: Tagging up ⭐ Eisenfaust ⭐ in Flussufer.
botgart_1  | 2020-10-30T17:54:00.308Z info [TS3Connection.js]: Tagging up Klingenschreck/Lukas in Flussufer.
botgart_1  | 2020-10-30T17:54:00.310Z info [TS3Connection.js]: Tagging up [MS] Alesia in Flussufer.
botgart_1  | 2020-10-30T17:54:00.978Z info [TS3Connection.js]: Tagging down [CS] Gunnar - Cat N Hemd in Flussufer.
botgart_1  | 2020-10-30T17:54:00.979Z error [TS3Connection.js]: Wanted to write raid for commander [CS] Gunnar - Cat N Hemd during tag-down, but no raid start was stored.
```

**Comment by Xyaren** [comment]
Most likely fixed by f3c5bf64a60c5aa09154361e6f55007d611bc671
Infinite loop blocked everything.
